staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -1444,6 +1445,15 @@ func TestSlowNodeSync(t *testing.T) {
 		t.Fatalf("error creating node3, err: %v", err)
 	}
 
+	// Allow a bit of time for the informer cache to get populated with the new
+	// node
+	if err := wait.PollUntilContextCancel(wait.ContextForChannel(stopCh), 10*time.Millisecond, true, func(ctx context.Context) (done bool, err error) {
```

Comment:
nit, this should be a context with timeout, stopCh is only closed when the test finish

Reply:
Are you OK to merge this and follow up with a timeout instead?